ledger/error/src/main/scala/com/daml/error/BaseError.scala

```diff
@@ -91,9 +91,14 @@ trait LocationMixin {
 
 object BaseError {
   private val ignoreFields = Set("cause", "throwable", "loggingContext")
-  val SecuritySensitiveMessageOnApi =
+  val SanitizedSecuritySensitiveMessageOnApiPrefix =
```

Comment:
What's wrong with the old name? Sanitized makes me think that we actually do some sanitization on some message, which we don't. We're just returning this message + correlation id

Reply:
I was interpreting `SecuritySensitiveMessageOnApi` as this specific message being sensitive, while I thought the sensitive one being the one that one that has been replaced by it.

Changing to `SecuritySensitiveMessageOnApiPrefix`